haskell/cabal
# `cabal check`: add `--ignore` option

Add `--ignore` option to `cabal check`, so you can ignore specific warnings. Closes #8587.

Include the following checklist in your PR:


---

**QA Notes**

- cd into a new folder
- `cabal init -nm`
- `cabal check`
- inspect the output
- `cabal check --ignore=no-category`
- the specific warning is gone.
